public/nativedisplay.h

```diff
@@ -82,6 +82,8 @@ class NativeDisplay {
   virtual void SetOutputBuffer(HWCNativeHandle /*buffer*/,
                                int32_t /*acquire_fence*/) {
   }
+  // Checks If the pixel format requested is supported or not
```

Comment:
Also, change it similar to this: https://github.com/01org/IA-Hardware-Composer/pull/30/commits/7ca578d29b73c7d28d2be4576160bb326e43dd1c#diff-c2573ac737057c13556ba84bd29291ceR81